ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+HCP_CLI="/usr/bin/hcp"
+
+MCE=${MCE_VERSION:-""}
+CLUSTER_NAME=$(echo -n "${PROW_JOB_ID}"|sha256sum|cut -c-20)
+if [[ -n ${MCE} ]] ; then
+    CLUSTER_NAMESPACE_PREFIX=local-cluster
+else
+    CLUSTER_NAMESPACE_PREFIX=clusters
+fi
+#CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE_PREFIX}-${CLUSTER_NAME}
+
+if [[ -n ${MCE} ]] ; then
+  if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    # shellcheck source=/dev/null
+    source "${SHARED_DIR}/proxy-conf.sh"
+  fi
```

Comment:
currently it's done here:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml#L20-L22

but not in https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml and I'm simply trying to refactor keeping the existing behaviours.

Reply:
```
if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
    # shellcheck source=/dev/null
    source "${SHARED_DIR}/proxy-conf.sh"
  fi
```
I think we shouldn't include source "${SHARED_DIR}/proxy-conf.sh" in the MCE judgment. It suits all relevant logic for BM, and it would be more appropriate to place it outside.

Reply:
done

Reply:
oh, you were suggesting to always execute it to prepare for the future.
Sorry, I misunderstood you